src/message_gems_parser.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+def get_gem_count_from_gem_string(message: str):
+    gem_template_string_anchor = '-gems'
```

Comment:
i thought we agreed je gems- ta e better?

Reply:
yes i misunderstood.. fixed in commit [13ca5bf](https://github.com/CraftsmenLtd/DiscordGems/pull/30/commits/13ca5bfb315f0b06ca2874c32a61aa1f32382005)